static/css/ProfilePage.css

```diff
@@ -133,8 +133,36 @@ header{
     padding: 20px;
     background-color: #fff;
     box-shadow: 0 0 10px rgba(0, 0, 0, 0.5);
+    border-radius: 1em 1em;
     z-index: 2;
 }
+.input_box{
```

Comment:
The #myModal modal seems well-styled, but adding some comments to explain its purpose or any specific styling choices that may not be immediately obvious would help provide clarity for anyone who works on it going forward